lib/src/widgets/chat/ds_image_message_bubble.dart

```diff
@@ -0,0 +1,179 @@
+                                            Icons.arrow_back_ios,
+                                            color: DSColors.neutralLightSnow,
+                                          ),
+                                        ),
+                                        Expanded(
+                                          child: ListTile(
+                                            contentPadding: EdgeInsets.zero,
+                                            leading: DSUserAvatar(
+                                              text: appBarText,
+                                            ),
+                                            title: DSHeadlineSmallText(
+                                              text: appBarText,
+                                              color: DSColors.neutralLightSnow,
+                                            ),
+                                          ),
+                                        ),
+                                      ],
+                                    ),
+                                  ),
+                                ),
+                              ),
+                              body: GestureDetector(
+                                onTap: () => _controller.showAppBar(),
+                                child: Container(
+                                  width: MediaQuery.of(context).size.width,
+                                  height: MediaQuery.of(context).size.height,
+                                  padding: const EdgeInsets.all(8.0),
+                                  color: Colors.black,
+                                  child: PinchZoom(
+                                    resetDuration:
+                                        const Duration(milliseconds: 100),
+                                    child: DSCachedNetworkImageView(
+                                      url: url,
+                                      placeholder: () {},
+                                      errorWidget: () {},
+                                      fix: BoxFit.contain,
+                                    ),
+                                  ),
+                                ),
+                              ),
+                            );
+                          },
+                        ),
+                      ),
+                    );
+                  },
+                );
+              }
+            },
+            child: DSCachedNetworkImageView(
+              width: 240.0,
+              height: 240.0,
+              url: url,
+              placeholder: (context, url) => const Padding(
+                padding: EdgeInsets.all(80.0),
+                child: CircularProgressIndicator(),
+              ),
+              errorWidget: (context, url, error) {
+                _controller.setError();
+                return Image.asset(
+                  'assets/images/file_image_broken.png',
+                  package: DSUtils.packageName,
```

Comment:
Now you can delete this "assets/images/file-image-broken.png" file from sample project since isn't necessary anymore.